webplugin/js/app/km-rich-text-event-handler.js

```diff
@@ -279,13 +279,13 @@ Kommunicate.richMsgEventHandler = {
         var target = e.target || e.srcElement;
         var requestType = target.dataset.requesttype;
         var buttonType = target.dataset.buttontype || target.type;
+        if(buttonType !="submit"){   
+            return ;
+        }
         var data = {};
         var form =target.parentElement.getElementsByClassName('km-btn-hidden-form')[0]
         var isActionableForm = (form.className.indexOf("mck-actionable-form") != -1 );
```

Comment:
@jithinbalu safe side, can we add a check for form exists?

Reply:
@devashishmamgain  changes are done